submissions/cluster.py

```diff
@@ -0,0 +1,197 @@
+import os
+import cv2
+import pytesseract
+import multiprocessing
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
let's not import matplotlib here, since we are not using it during server requests, right? Let's move it inside the plotting routine.